conduct/conduct_enforcement.md

```diff
@@ -5,6 +5,13 @@ Committee. It's used when we respond to an issue to make sure we're consistent
 and fair. It should be considered an internal document, but we're publishing it
 publicly in the interests of transparency.
 
+Enforcing the code of conduct with punitive measures or reprimands is an
```

Comment:
Thanks folks! Will use this as I try to merge/respond to all feedback in the main PR. I hope I won't bungle the "staged merge" :)